src/main.rs

```diff
@@ -9,7 +9,7 @@ use std::env;
 use structopt::StructOpt;
 
 fn main() -> Result<()> {
-    let our_rust_log = "cranelift_wasm=warn,cranelift_codegen=info,wasi_common=info";
+    let our_rust_log = "cranelift_wasm=warn,cranelift_codegen=error,wasi_common=error";
```

Comment:
FWIW, I'd leave `wasi_common` at `info` as it won't be that verbose so it shouldn't blow the payload size, especially after we've increased the max limit. However, decreasing `cranelift_codegen` to `error` is a good idea :+1:

```suggestion
    let our_rust_log = "cranelift_wasm=warn,cranelift_codegen=error,wasi_common=info";
```